GitVersion.yml

```diff
@@ -2,6 +2,4 @@ assembly-versioning-scheme: Major
 next-version: 6.0
 branches:
   develop:
-    tag: alpha
-  releases?[/-]:
```

Comment:
sorry andreas. i didnt need to know for the changes i am doing so didnt bother looking it up